test/cppsuite/src/main/database_operation.cpp

```diff
@@ -125,6 +127,19 @@ database_operation::populate(
     logger::log_msg(LOG_INFO, "Populate: finished.");
 }
 
+void
+database_operation::checkpoint_operation(thread_context *tc)
+{
+    logger::log_msg(
+      LOG_INFO, type_string(tc->type) + " thread {" + std::to_string(tc->id) + "} commencing.");
+
+    while (tc->running()) {
+        std::cout << "Doing checkpoint" << std::endl;
```

Comment:
Can you change this to use the logger and then drop the `#include <iostream>` above?

Reply:
Yes! See e3243ec508b734ebabdbb51fbdb7c9568f3f986c

Reply:
Thanks! Looks like we've dropped the `std::cout << "Doing checkpoint" << std::endl;` line but haven't replaced it with a logger call. Is this intentional?

Reply:
Yes it was for debugging purposes, sorry.